### PR TITLE
Add KMS Key With Vulnerable Policy query #1257

### LINKED ISSUE #1257: Add KMS Key With Vulnerable Policy query for Ansible
### Platform
Ansible

### Description
Checks if the policy is vulnerable and needs updating.

### PR DESCRIPTION
**Pull request by joelcarvalhocheckmarx**
Checks if the policy is vulnerable and needs updating.

Closes #1257

### CODE REVIEW

File: assets/queries/ansible/aws/kms_key_with_vulnerable_policy/query.rego [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  statement = aws_kms.policy.Statement[_]
+  check_permission(statement) == true
+   
+    result := {
+            "documentId": document.id,
+            "searchKey": sprintf("name=%s.{{community.aws.aws_kms}}.policy", [task.name]),
+            "issueType": "IncorrectValue",
+            "keyExpectedValue": sprintf("name=%s.{{community.aws.aws_kms}}.policy is correct", [task.name]),
+            "keyActualValue": sprintf("name=%s.{{community.aws.aws_kms}}.policy is incorrect", [task.name]),
```
Comment (joaomartinscx) [inline assets/queries/ansible/aws/kms_key_with_vulnerable_policy/query.rego:18]: The correctness of the policy here is not clear, could you specify what is or is not expected in the resource?

**Review by joaomartinscx — CHANGES_REQUESTED** [review]

**Comment by markmishaevcx** [comment]
Scan submitted to Checkmarx

**Comment by markmishaevcx** [comment]
Logo<br />**Checkmarx SAST - Scan Summary & Details**
### Cx-SAST Summary
Total of 4 vulnerabilities<br />High&nbsp;**0 High**<br />Medium&nbsp;**3 Medium**<br />Low&nbsp;**1 Low**<br />Info&nbsp;**0 Info**<br />
### Violation Summary
Medium&nbsp;**3** **Medium**<br />
[View more details on Checkmarx UI](https://sast.checkmarx.net/CxWebClient/ViewerMain.aspx?scanid=1049089&projectid=736)
### Cx-SAST Details
|Lines|Severity|Category|File|Link|
---|---|---|---|---
[30](https://github.com/Checkmarx/kics/blob/feature/add_query_ansible_kms_key_with_vulnerable_policy/cmd/builder/main.go#L30) |Medium|Path_Traversal|cmd/builder/main.go|[Checkmarx](https://sast.checkmarx.net/CxWebClient/ViewerMain.aspx?scanid=1049089&projectid=736&pathid=3)
[52](https://github.com/Checkmarx/kics/blob/feature/add_query_ansible_kms_key_with_vulnerable_policy/pkg/kics/service.go#L52) |Medium|Denial_Of_Service_Resource_Exhaustion|pkg/kics/service.go|[Checkmarx](https://sast.checkmarx.net/CxWebClient/ViewerMain.aspx?scanid=1049089&projectid=736&pathid=1)
[82](https://github.com/Checkmarx/kics/blob/feature/add_query_ansible_kms_key_with_vulnerable_policy/test/main_test.go#L82) |Medium|Denial_Of_Service_Resource_Exhaustion|test/main_test.go|[Checkmarx](https://sast.checkmarx.net/CxWebClient/ViewerMain.aspx?scanid=1049089&projectid=736&pathid=2)

**Review by joaomartinscx — APPROVED** [review]
Everything looks okay, good job!

**Pull request merged**